v2/card_pools/startup.json

```diff
@@ -30,12 +30,7 @@
     "id": "startup_ashes_plus_midnight_sun",
     "format_id": "startup",
     "name": "Startup (Ashes + Midnight Sun)",
-    "card_cycle_ids": [
-      "ashes",
-      "borealis",
-      "system_gateway",
-      "system_update_2021"
-    ],
+    "card_cycle_ids": ["ashes", "system_gateway", "system_update_2021"],
```

Comment:
This shouldn't change, unless we want the intermediate versions of Startup between full releases (i.e. the versions of Startup containing 1.5 cycles) to be represented temporarily.

Reply:
this is the same issue as the other comment.  If we kept borealis, we would be *forced* to have parhelion in here as well.  Removing borealis lets us keep just the one set in card_set_ids without any failing tests.